styles/styles.css

```diff
@@ -0,0 +1,127 @@
+* {
+  font-family: "Open Sans", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.title-small { 
+  display: block;
+  font-size: 1.10em;
+  margin-top: 1em;
+  margin-bottom: 1em;
+  margin-left: 0;
+  margin-right: 0;
+  font-weight: bold;
+}
+
+.header {
+  align-items: center;
+  background-color: #FFF;
+  border-top: 3px solid #F5C85F;
+  display: flex;
+  justify-content: center;
+  padding: 20px 0 40px;
+  width: 100%;
+}
+
+.vertical-separator {
+  background: linear-gradient(#FFF, #F5C85F, #FFF);
+  width: 3px;
+  height: 300px;
+  margin: 0px 30px;
+} 
+
+.features::before {
+  content:"";
+  background-color: #FFF;
+  background-image: url("../resources/triangle.png");
+  background-repeat: repeat-x;
```

Comment:
I think there's no need for this, it will repeat itself

Reply:
BTW, how do you position this pseudo-element? :thinking:

Reply:
hmm, its an empty element that goes before features and occupies all the width of its parent, unless im getting something wrong?